include/caf/click_detector.h

```diff
@@ -7,6 +7,9 @@
 #ifndef _CLICK_DETECTOR_H_
 #define _CLICK_DETECTOR_H_
 
+#include <stdint.h>
```

Comment:
You need to split commit message line to fix the newly introduced compliance error: `Commit message body line exceeds max length (219>72):`